operations/helm/charts/mimir-distributed/capped-large.yaml

```diff
@@ -60,12 +60,21 @@ ingester:
     limits:
       cpu: 4
       memory: 15Gi
+  topologySpreadConstraints: {}
```

Comment:
no-op. I just want to note, that for multi zone, the anti affinity will be between zones, so if topologySpreadConstraints is `{}` then it will allow any number of ingesters on the same node as long as they are of the same zone. I'll have to document this in the migration doc as well to be clear.

Reply:
not sure if you are planning on having `multi-zone.yaml` values file (same concept as `small.yaml`). But you can add the `topologySpreadConstraints` there.

Reply:
I think it's fine, we use multi zone ingesters/store-gateways without topology constraints, so no need to add extra files.